config/variants/multi-gw-postgres/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kong
+
+resources:
+- ../postgres
+- gateway_admin_service.yaml
+- gateway_deployment.yaml
+
+components:
+  - ../../image/oss
+
+patchesStrategicMerge:
```

Comment:
`patchesStrategicMerge` is deprecated in kustomize v5: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv5.0.0

We should now use `patches`. You can find some inspiration in this PR description: https://github.com/kubernetes-sigs/kustomize/pull/4911